linux_os/guide/services/ssh/ssh_server/disable_host_auth/rule.yml

```diff
@@ -7,9 +7,17 @@ description: |-
     more secure than <tt>.rhosts</tt> authentication. However, it is
     not recommended that hosts unilaterally trust one another, even
     within an organization.
-    <br /><br />
-    To disable host-based authentication, add or correct the
-    following line in <tt>/etc/ssh/sshd_config</tt>:
+    <br />
+    The default SSH configuration disables host-based authentication. The appropriate
+    configuration is used if no value is set for <tt>HostbasedAuthentication</tt>.
+    <br />
+    To explicitly disable host-based authentication, add or correct the
+    following line in
+    {{% if sshd_distributed_config == "true" %}}
+    <tt>/etc/ssh/sshd_config.d/00-complianceascode-hardening.conf</tt>:
+    {{% else %}}
+    <tt>/etc/ssh/sshd_config</tt>:
+    {{% endif %}}
```

Comment:
This `if` block repeats often in many rules. Would it be better to have a macro that would contain this block?

Reply:
I'm okay converting this to a macro, my only concern would be readability of the source code...

Reply:
Macro added in e7efd94f67021da6b54749a1eea4f83aeac977ce